CI.md

```diff
@@ -8,6 +8,6 @@ GitHub Actions are used to run the Continuous Integration for amazon.aws collect
 | ------ |-------| ------ | -----------|
 | changelog |Checks for the presence of Changelog fragments | 3.9 | devel |
 | Linters | Runs `black` and `flake8` on plugins and tests | 3.9 | devel |
-| Sanity | Runs ansible sanity checks | 3.8, 3.9, 3.10, 3.11 | Stable-2.12, 2.13, 2.14 (not on py 3.11), Stable-2.15+ (not on 3.8) |
-| Unit tests | Executes the unit test cases | 3.9, 3.10 | Stable-2.12+ |
+| Sanity | Runs ansible sanity checks | 3.10, 3.13 | devel, milestone, stable-2.16, stable-2.17 (Python 3.13), stable-2.18, stable-2.19 |
+| Unit tests | Executes the unit test cases | 3.10, 3.13 | evel, milestone, stable-2.16, stable-2.17 (Python 3.13), stable-2.18, stable-2.19 |
```

Comment:
I know there's been some back and forth on the release PR, and ideally, this change shouldn't be part of it. However, to avoid further delays in the release, let's proceed with it for now. That said, the current values aren't accurate . You can refer to the versions used in the test runs [here](https://github.com/ansible-collections/amazon.aws/actions/runs/15004186618). Once the check job passes, please remove the dependent Zuul PR and `TestAllTheTargets`, and update this file accordingly before merging.